### ISSUE TITLE
Talkback for android announces ‘double tap twice to activate’ for headers and subheaders

### ISSUE THREAD

**Issue by Samerabutt** [facebook/react#14886]
**Do you want to request a *feature* or report a *bug*?**
 Bug

**What is the current behavior?**
When the user uses the swipe mechanism to navigate through fields, talkback for android announces ‘double tap twice to activate’ when focusing on headers and sub headers

**If the current behavior is a bug, please provide the steps to reproduce and if possible a minimal demo of the problem. Your bug will get fixed much faster if we can run your code and it doesn't have dependencies other than React. Paste the link to your JSFiddle (https://jsfiddle.net/Luktwrdm/) or CodeSandbox (https://codesandbox.io/s/new) example below:**

1. I created an ap using create-react-app
2. I use talkback for android, chrome browser, to access the app and use the swipe mechanism to navigate through the fields
3. When headers and subheaders are in focus, talkback announces ‘double tap to activate’

**What is the expected behavior?**
Talkback should not announce ‘double tap to activate’ as the headers and sub headers can not be activated.

**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**
React:16.8.1
OS: Oreo 8.1/pie 9
Browser: chrome
Device: Samsung edge

**Comment by matheusmonte** [comment]
@gaearon @Samerabutt Can I get this issue to investigation ?

**Comment by gaearon** [comment]
This has no actionable details. No CodeSandbox, no investigation of whether plain HTML has the same behavior. So this isn't an issue we can help you with.
